package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] I couldn't run your app from my local machine because the Start script is missing in your package.json file. Please fix this 👍🏻